include/hpp/pinocchio/serialization.hh

```diff
@@ -89,4 +93,187 @@ inline void serialize(Archive & ar, Eigen::Array<_Scalar,_Rows,_Cols,_Options,_M
 } // namespace serialization
 } // namespace boost
 
+namespace hpp {
+namespace serialization {
+namespace remove_duplicate {
+template<typename Key, typename Compare = std::less<Key> >
+struct ptr_less : Compare {
+  inline bool operator() (Key const* t1, Key const* t2) { return Compare::operator() (*t1, *t2); }
+};
+
+template<typename Derived>
+struct eigen_compare {
+  bool operator() (const Eigen::DenseBase<Derived>& a,
+                   const Eigen::DenseBase<Derived>& b)
+  {
+    if (a.size() < b.size()) return true;
+    if (a.size() > b.size()) return false;
+    for(Eigen::Index i = 0; i < a.size(); ++i) {
+      if (a.derived().data()[i] < b.derived().data()[i]) return true;
```

Comment:
It should be PlainObjectBase if you refer to data ;)

Reply:
Done. Thanks.